lib/ukvmem/arch/arm/pagefault64.c

```diff
@@ -39,10 +40,13 @@ static int vmem_arch_pagefault(void *data)
 
 	rc = vmem_pagefault(vaddr, faulttype, ctx->regs);
 	if (unlikely(rc < 0)) {
-		uk_pr_crit("Cannot handle %s page fault at 0x%"__PRIvaddr
-			   " (ec: 0x%lx): %d\n",
-			   faultstr[faulttype & UK_VMA_FAULT_ACCESSTYPE],
-			   vaddr, ctx->esr, rc);
+		vas = uk_vas_get_active();
+		if (unlikely(vas && !(vas->flags & UK_VAS_FLAG_NO_PAGING)))
+			uk_pr_crit("Cannot handle %s page fault at 0x%"
+				   __PRIvaddr" (ec: 0x%x): %d\n",
+				   faultstr[faulttype &
+					    UK_VMA_FAULT_ACCESSTYPE],
+				   vaddr, ctx->error_code, rc);
```

Comment:
Note to self: instead of printing `rc`, print its `strerror(rc)` instead.

Reply:
Do both, strerror plus rc